awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -258,15 +261,21 @@ class JobTemplateForm extends Component {
                   />
                 )}
               />
-              <FormField
-                id="template-project"
+              <Field
                 name="project"
-                type="number"
-                label={i18n._(t`Project`)}
-                tooltip={i18n._(t`Select the project containing the playbook
-                you want this job to execute.`)}
-                isRequired
                 validate={required(null, i18n)}
+                render={({ form }) => (
+                  <ProjectLookup
+                    value={project}
+                    tooltip={i18n._(t`Select the project containing the playbook
+                    you want this job to execute.`)}
+                    onChange={value => {
+                      form.setFieldValue('project', value.id);
+                      this.setState({ project: value });
```

Comment:
when playbook select is implemented in the near future, we'll want to make sure we clear / reset the value when projects change.

Reply:
Cool. I'll include this when I add the playbook select.